pb/contacts_test.go

```diff
@@ -0,0 +1,52 @@
+package pb_test
+
+import (
+	"testing"
+
+	. "github.com/kansaslabs/rossby/pb"
+	"github.com/stretchr/testify/require"
+)
+
+func TestContactValidate(t *testing.T) {
+	var phonetests = []struct {
+		contact string
+		valid   bool
+	}{
+		{"+14155552671", true},
+		{"+442071838750", true},
+		{"+551155256325", true},
+		{"foo", false},
```

Comment:
Do we need to test too few/too many numbers?

Reply:
Yes! Good idea, thanks!